libs/template/presets/storefront/experience/footer.ts

```diff
@@ -1,17 +1,29 @@
 import { ExperienceComponent } from '@spryker-oryx/experience';
 import { IconTypes } from '@spryker-oryx/ui/icon';
+import { featureVersion, i18n } from '@spryker-oryx/utilities';
 
 const image = (graphic: string) => ({
   type: 'oryx-content-image',
   content: { data: { graphic, alt: graphic } },
 });
 
-const heading = (text: string, rule = {}) => ({
+const heading = (token: string, rule = {}) => ({
   type: 'oryx-content-text',
-  content: { data: { text: `<b>${text}</b>` } },
+  content: {
+    data: {
+      text: `${text(token)}`,
+    },
+  },
   options: { rules: [rule] },
 });
 
+const text = (token: string): string => {
+  const text = i18n(token);
```

Comment:
I don't think this will work besides auto converting tokens to string repersentation. It's not reactive either, so only first/default language will land in sample data. Thus not much point for wiring i18n here IMO.

Reply:
agreed. Also we want those footer navigations to be loaded from CMS anyway.